board/safety/safety_toyota.h

```diff
@@ -190,6 +191,13 @@ static void toyota_rx_hook(const CANPacket_t *to_push) {
       }
     }
 
+    // wrap lateral controls on main
+    if (addr == 0x1D3) {
```

Comment:
why do we need this?

Reply:
To check the status of mainon. We only want to allow Break hold when mainon is active